src/lib/edge-functions/registry.ts

```diff
@@ -33,18 +38,19 @@ interface EdgeFunctionsRegistryOptions {
   directories: string[]
   env: Record<string, { sources: string[]; value: string }>
   getUpdatedConfig: () => Promise<Config>
-  internalDirectories: string[]
-  internalFunctions: Declaration[]
   projectDir: string
   runIsolate: RunIsolate
   servePath: string
+  importMapFromTOML?: string
 }
 
 export class EdgeFunctionsRegistry {
   private buildError: Error | null = null
   private bundler: typeof import('@netlify/edge-bundler')
   private configPath: string
-  private declarationsFromDeployConfig: Declaration[]
+  public importMapFromDeployConfig?: string
```

Comment:
NIT: the public keyword is not needed.

```suggestion
 importMapFromDeployConfig?: string
```

Reply:
Agree it's not needed, but I prefer to have this here - makes it obvious that it's accessed from other places.